backend/src/project/project.controller.ts

```diff
@@ -22,7 +22,11 @@ export class ProjectController {
 
   @Post()
   @HttpCode(200)
-  @Auth(Role.projectManager, Role.founder)
```

Comment:
I would not go with this until the Kali integration is done. 
It would be safer to have "global roles" to handle the initial creation (eg. sporos founder or platform managers) for the beta testers. 

Per-DAO permission are going to be implemented in #19 and change a bit the current take.

To recap, the final design would be to have a layer for "core users" with roles (eg. Sporos DAO admin/PM) to manage at platform level (the Sweat Token platform) and another layer for DAO "members" to  handle per-DAO access control and roles definition (founder, PM, ..)

Reply:
Let's keep this, will refator in the woriking branch!